### PR TITLE
Add an explanation of root and reference sequences

### PR DESCRIPTION
**Pull request by kimandrews**
## Description of proposed changes

Adds a doc explaining how the terms "root sequence" and "reference sequence" are used in Nextstrain. Inclusion of this doc was motivated by confusion observed regarding these terms. The doc is added to be located immediately following an already existing doc describing how to choose a reference sequences.

## Checklist

- [ ] Checks pass

### CODE REVIEW

**Comment by jameshadfield** [comment]
[Preview page](https://nextstrain--227.org.readthedocs.build/en/227/guides/bioinformatics/root-and-ref-seqs.html)

File: src/guides/bioinformatics/root-and-ref-seqs.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+
+What can go wrong?
+==================
+
+When creating standard Nextclade datasets, a common mistake is forgetting to add the reference sequence to the tree as a parent node using the ``--root-sequence`` option in ``augur ancestral``. If you forget to do this, the mutational differences between the parent and root node sequences will not be encoded in the tree. Since Nextclade assumes that the reference sequence in your FASTA file is the parent node (which is effectively the root node of the encoded tree), the program will notice that the mutation calls on the branches don't match up with the reference sequence. This is because mutation calls include specific information about the reference sequence; they are defined in the format “T36G”, in which “T” is expected to be the nucleotide in the reference sequence at position 36. If those mutation calls don't match up with the reference sequence (e.g., the reference sequence does not have a “T” at position 36), Nextclade will produce an error message::
+
+   "Please check that your reference tree is consistent with your reference sequence."
+
+
```
Comment (genehack) [inline src/guides/bioinformatics/root-and-ref-seqs.rst:35]:
```suggestion
```

**Review by genehack — APPROVED** [review]

**Review by jameshadfield — APPROVED** [review]
Let's merge!

File: src/guides/bioinformatics/root-and-ref-seqs.rst (hunk 1 above)
Comment (kimandrews) [reply src/guides/bioinformatics/root-and-ref-seqs.rst]: Done in [0529b12](https://github.com/nextstrain/docs.nextstrain.org/pull/227/commits/0529b125817c1f7772ea7b0d6533fc8668bb74aa)

**Pull request merged**